hypothesis-python/tests/cover/test_health_checks.py

```diff
@@ -28,26 +28,28 @@
 
 from tests.common.utils import no_shrink
 
-HEALTH_CHECK_SETTINGS = settings(max_examples=11, database=None)
+HEALTH_CHECK_SETTINGS = settings(
+    max_examples=11, database=None, suppress_health_check=()
+)
 
 
 def test_slow_generation_fails_a_health_check():
-    @HEALTH_CHECK_SETTINGS
+    @settings(HEALTH_CHECK_SETTINGS, deadline=200)
```

Comment:
default deadline is already 200, right? are we overriding this with a profile somewhere I'm forgetting?

Reply:
That's the default deadline, but we override it with the crosshair profile in #4034, and so explicitly specifying it here makes the tests check what they mean to regardless of the global config.

Reply:
ah, of course.